src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -806,6 +804,10 @@ def computer_export_config(computer, output_file, user, sort):
         )
     try:
         computer_configuration = computer.get_configuration(user)
+        if output_file is None:
+            output_file = pathlib.Path(f'{computer.label}-config.yml')
```

Comment:
You are not actually testing the case of the default filename. Please add this test for all commands

Reply:
Ah, yes. Forgot to test this for the `Computer`.